docs/CONTRIBUTING.md

```diff
@@ -20,8 +20,9 @@ Here are some tips to make sure your Pull Request (PR) can be merged smoothly:
 ### Pre-requisites
 
 * Install [Git](https://git-scm.com/downloads)
-* Install [NodeJS](https://nodejs.org/).
-* Install [Yarn](https://yarnpkg.com/en/docs/install). Use Yarn 1.x as DIM is not guaranteed to be compatible with later versions of Yarn. If you're used to NPM, see "[Migrating from NPM](https://yarnpkg.com/lang/en/docs/migrating-from-npm/)".
+* Install [NodeJS](https://nodejs.org/)
+* Install [Yarn](https://yarnpkg.com/en/docs/install)
+  * Use Yarn 1.x as DIM is not guaranteed to be compatible with later versions of Yarn. If you're used to NPM, see "[Migrating from NPM](https://yarnpkg.com/lang/en/docs/migrating-from-npm/)".
```

Comment:
Aren't these inconsistent now? Some have a period at the end and some don't.

Reply:
Looking at line 22, it seemed it made sense for the Install steps to be without punctuation since they're not full sentences. This is also why I split the Yarn explanation on line 25. Ideally, it creates a visual separation between the more basic steps of installing Git, Node, and Yarn and the following explanations.